health/mapping/es.py

```diff
@@ -76,11 +76,34 @@
 
 
 def init_elastic(es, index_to_create="ms_health_idx_1"):
-    r = requests.get("%s/%s" % (es, "ms_health"))
-    if r.status_code == 200:
-        print("Index ms_health already exists, nothing to do!")
+    # TODO(kzaitsev): replace prints with logging
+    r = requests.get("%s/%s" % (es, index_to_create))
+    if r.ok:
+        print("Index {} already exists, nothing to do!".format(
+            index_to_create))
     else:
-        requests.post("%s/%s" % (es, index_to_create),
-                      data=json.dumps(mapping))
-        if r.status_code == 200:
-            print("Index %s created successfully" % index_to_create)
+        r = requests.put("%s/%s" % (es, index_to_create),
+                         data=json.dumps(mapping))
+        if r.ok:
+            print("Index {} created successfully".format(index_to_create))
+        else:
+            print("Got {} status when creating index {}. {}".format(
+                r.status_code, index_to_create, r.text))
+
+    # enable fielddata for some Logger and http_status:
+    for field in ['Logger', 'http_status']:
+        data = {
```

Comment:
This is an update to already existing index, to enable fielddata for certain fields. The request will use `_mapping` api, i.e. `/log-*/_mapping/log` Probably we can change it to keyword =/